src/Controller/HearingController.php

```diff
@@ -73,15 +74,19 @@ public function hearing(CaseEntity $case, HearingPostRepository $hearingPostRepo
 
         // Detect whether most recent hearing post has been forwarded or even exists
         $mostRecentPost = reset($hearingPosts);
-        $hasNewUnforwardedPost = $mostRecentPost && !$mostRecentPost->getForwardedOn();
+        if ($mostRecentPost instanceof HearingPostResponse) {
+            $requiresProcessing = $mostRecentPost && !$mostRecentPost->getApprovedOn();
+        } else {
+            $requiresProcessing = $mostRecentPost && !$mostRecentPost->getForwardedOn();
+        }
```

Comment:
`$requiresProcessing = … ? … : …;`

Reply:
Fixed in ae50d3f482f55c5fb7a2eed4bcc8af9b9388689a.